src/DynamoPackages/PackageLoader.cs

```diff
@@ -503,7 +500,19 @@ private void ScanPackageDirectories(string root, IPreferences preferences)
 
                     var pkg = ScanPackageDirectory(dir, checkCertificates);
                     if (pkg != null && preferences.PackageDirectoriesToUninstall.Contains(dir))
```

Comment:
Something is strange here. `PackageDirectoriesToUninstall` is always empty in my testing. Even when there is a package path in the settings file. Any ideas?

Reply:
FYI @pinzart90

Reply:
You mean the value of `PackageDirectoriesToUninstall ` when debugging ?
Or the UI combobox that corresponds to that data ?

Reply:
When debugging. I am trying to get a single built-in package set to `Unloaded` after restart. It always show up as `Loaded` for me after restart (and the path are removed from the settings file).

Reply:
Looks like the directories are removed here: https://github.com/DynamoDS/Dynamo/pull/11917/files#diff-7ae21a0e381958737cc115ada84cbd9f96efe3315928cbc44b367967923cf55bR804 before we get to this spot.

Reply:
found the issue...I was resetting the `PackageDirectoriesToUninstall` for built in packages.
SHould work now...but there is still a bit of UX to do I guess...there is no clear indication when a built in package is manually unloaded (will be unloaded until explicitly Loaded) or automatically unloaded due to a conflict (which will be unloaded only as long as the conflict persists)

Reply:
Thanks, I will give it a try. For the UX part, I have not found it that confusing. I will think about it when I play around with it today.

Reply:
@pinzart90 Works fine now!